node_package/src/clientStartup.ts

```diff
@@ -213,6 +211,7 @@ function unmount(el: Element): void {
   const domNode = document.getElementById(domNodeId);
   if(domNode === null){return;}
   try {
+    // Might need updating? https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html
```

Comment:
`unmountComponentAtNode` is considered legacy https://reactjs.org/docs/react-dom.html#unmountcomponentatnode but would need to store references to root instead if we were to update

Reply:
Please see #1466.